### DISCUSSION TITLE
Contributing document

### DISCUSSION THREAD

**Discussion by YosypC** [flxzt/rnote#851] [Development]
A contributing file is missing. Any plans of making one? If would speed development up.

**Comment by flxzt** [comment]
Do you have suggestions what to include in such a file?

**Reply by YosypC** [reply]
Not quite sure.
I'm not new to FOSS but I haven't really contributed with code, although I've found some [resources to share](https://opensource.guide/how-to-contribute/#anatomy-of-an-open-source-project):

"CONTRIBUTING: Whereas READMEs help people use the project, contributing docs help people contribute to the project. It explains what types of contributions are needed and how the process works. While not every project has a CONTRIBUTING file, its presence signals that this is a welcoming project to contribute to. A good example of an effective Contributing Guide would be the one from [Codecademy’s Docs repository](https://www.codecademy.com/resources/docs/contribution-guide)."

Also have a look at this:

[Writing your contributing guidelines](https://opensource.guide/starting-a-project/#writing-your-contributing-guidelines)

**Comment by flxzt** [comment]
Let's continue this discussion in #876
